src/ontology/cl-edit.owl

```diff
@@ -23391,6 +23392,16 @@ SubClassOf(obo:CL_0011029 obo:CL_0000151)
 SubClassOf(obo:CL_0011029 ObjectSomeValuesFrom(obo:BFO_0000051 obo:GO_0042151))
 SubClassOf(obo:CL_0011029 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_6073))
 
+# Class: obo:CL_0011031 (monocycte-derived dendritic cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32733468") obo:IAO_0000115 obo:CL_0011031 "A dendritic cell subset that develops from monocytes monocytes upon stimulation in the circulation that are involved in inflammation and infection.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_0011031 <http://orcid.org/0000-0001-5208-3432>)
```

Comment:
Def has typos and is hard to follow. 
What is circulating? Presumably these cells are not involved in infection but it fighting it?
Does this work?
"A dendritic cell that develops from a monocyte.  These cells are involved in inflammation in response to infection"

Reply:
Sure, I'll update this, then merge it. thanks!

Reply:
Also the label has a typo: 'monocycte-derived dendritic cell'  monocyte is spelled wrong

Reply:
fixed!

Reply:
the label and text def scream out design pattern